File: go-ut-bench/datasets/java/java_code_files_self_contained/boundary/boundary_000.java
class RangeValidator {
    public static int clamp(int value, int minVal, int maxVal) {
        if (minVal > maxVal) {
            throw new IllegalArgumentException("minVal must be less than or equal to maxVal");
        }
        if (value < minVal) {
            return minVal;
        }
        if (value > maxVal) {
            return maxVal;
        }
        return value;
    }
}

File: go-ut-bench/datasets/python/python_code_files_self_contained/boundary/boundary_000.py
def clamp(value: int, min_val: int, max_val: int) -> int:
    if min_val > max_val:
        raise ValueError("min_val must be less than or equal to max_val")
    if value < min_val:
        return min_val
    if value > max_val:
        return max_val
    return value